phaser/pipeline.py

```diff
@@ -129,8 +153,51 @@ def run(self):
         next_source = self.source
         for phase in self.phase_instances:
             destination = self.get_destination(phase)
-            phase.run(source=next_source, destination=destination)
-            next_source = destination  # for next phase in chain
+            self.run_phase(phase, next_source, destination)
+            next_source = destination
+
+    def run_phase(self, phase, source, destination):
+        logger.info(f"Loading input from {source} for {phase.name}")
+        data = self.load(source)
```

Comment:
If `self.load(source)` returns `list(dict)` from `read_csv(source).to_dict('records')`, then passing that `data` to `phase.load_data(data)` won't work for a `DataFramePhase`, because it expects a `DataFrame` to be passed in.

Need a test to validate my assertion and ensure it continues to work in the future.

Reply:
I added a test and a fix in a4fd662